.github/workflows/schedule-stale.yml

```diff
@@ -0,0 +1,52 @@
+on:
+  schedule:
+    - cron: "0 */12 * * *"
+name: Stale Bot workflow
+jobs:
+  build:
+    name: stale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

Comment:
Is checkout actually needed?

Reply:
Nope!
https://help.github.com/en/github/automating-your-workflow-with-github-actions/configuring-a-workflow#using-the-checkout-action